submission_rules.adoc

```diff
@@ -21,70 +21,91 @@ Unless otherwise stated all rules apply to closed and open.
 
 ## Review committee
 
-The MLPerf submission, review, and publication process is overseen by a review committee. 
+The MLPerf review and publication process is overseen by a review committee. 
 
 
 ### Structure
 
 The review committee consists of the following people:
 
+*   The relevant working group chairs
+*   The relevant power working group chairs
+*   Optionally, the review chair of relevant working group
+*   The executive director
+*   The president of the board
+*   One representative of each submitter
```

Comment:
I think we have multiple representatives from one submitters before. I addition to "One representative of each submitter.", I recommend we allow up to 3 observers from each submitter for providing clarifications in the review process.

Reply:
Done.